src/common/low_precision_transformations/src/rt_info/precisions_attribute.cpp

```diff
@@ -27,7 +25,8 @@ ov::Any PrecisionsAttribute::create(
     const std::shared_ptr<ngraph::Node>& node,
     const AttributeParameters& params) {
     auto& rt = ov::is_type<opset1::FakeQuantize>(node) ? node->output(0).get_rt_info() : node->get_rt_info();
-    return (rt[PrecisionsAttribute::get_type_info_static()] = PrecisionsAttribute());
+    return (rt[PrecisionsAttribute::get_type_info_static()] = PrecisionsAttribute(
+        ngraph::pass::low_precision::LayerTransformation::getDefaultPrecisions()));
```

Comment:
It's not good idea to execute `ngraph::pass::low_precision::LayerTransformation::getDefaultPrecisions()` in attribute. To pass precisions as input argument is better. Will be fixed later.